demos/dlgr/demos/chatroom/config.txt

```diff
@@ -21,12 +21,12 @@ browser_exclude_rule = MSIE, mobile, tablet
 
 [Database]
 database_url = postgresql://postgres@localhost/dallinger
-database_size = hobby-dev
+database_size = standard-0
```

Comment:
Was it necessary to use standard-0? did it not work on hobby-dev?
The hobby tier does not cost anything as far as I recall, so if someone does want to or have the means to pay, they can still use the demo knowing they wont be charged (even though they have to provide a credit card, in order to use redis in all dallinger demos)

Reply:
I think hobby-dev is free, standard-0 you have to pay for.
I do not see any indication of payment requirements for hobby-dev, standard costs $50/month
https://devcenter.heroku.com/articles/heroku-postgres-plans#standard-tier